README.md

```diff
@@ -10,7 +10,7 @@ The Jaeger Operator is an implementation of a [Kubernetes Operator](https://kube
 To install the operator, run:
 ```
 kubectl create namespace observability
-kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
```

Comment:
This will probably break clients/tutorials.  Why is this needed?

Reply:
We could probably keep the old one there. Unfortunately, this is a breaking change from the Operator SDK itself and some commands from it expect the file to be in this new location.